src/run.rs

```diff
@@ -423,7 +423,10 @@ pub fn run() {
     evaluate: matches.is_present("EVALUATE"),
     highlight: matches.is_present("HIGHLIGHT"),
     quiet: matches.is_present("QUIET"),
-    shell: matches.value_of("SHELL").unwrap(),
+    shell: matches
+      .value_of("SHELL")
+      .map(|s| Shell::new(s))
+      .unwrap_or(DEFAULT_SHELL),
```

Comment:
This should never happen, we have the `default_value`, I can restore the `unwrap()` we had before.